agent/agent.go

```diff
@@ -125,7 +126,15 @@ func NewAgent(options ...Option) (*Agent, error) {
 	configProfile := GetConfigCurrentProfile()
 
 	if agent.apiKey == "" {
-		if apikey, set := os.LookupEnv("SCOPE_APIKEY"); set && apikey != "" {
+		if dsn, set := os.LookupEnv("SCOPE_DSN"); set && dsn != "" {
+			dsnApiKey, dsnApiEndpoint, dsnErr := parseDSN(dsn)
+			if dsnErr != nil {
+				agent.logger.Printf("Error parsing dsn value: %v\n", dsnErr)
```

Comment:
```suggestion
				agent.logger.Printf("Error parsing dsn value: %v", dsnErr)
```

`log.Printf` does not need `\n`